### ISSUE TITLE
Delete button in the list selection window has the text "delet" in it

### ISSUE THREAD

**Issue by boredompwndu** [baranidlo/bench#86]
![image](https://user-images.githubusercontent.com/8730661/37098015-cbaee36a-21ea-11e8-852a-6c1a9f077c15.png)

**Comment by Sleet01** [comment]
This might be an issue with the recent German translation check-ins.  @cbmainz, do you want to look at this?

**Comment by cbmainz** [comment]
Yes, sorry, I will fix it. It has also be fixed in the other languages. Hope I get this right.

**Comment by Sleet01** [comment]
No problem!  This is how modern software development works! :)

**Comment by cbmainz** [comment]
I just corrected the english spelling of "Delete" in all other languages. I'm not able to to the translation.

I added some more classes in main.html before so the text can be translated now. But the translation has to be done by people who speak the language.
